priv/migrations/add_railway_ipc_published_messages.rb

```diff
@@ -0,0 +1,18 @@
+class AddRailwayIpcPublishedMessages < ActiveRecord::Migration
+  def change
+    create_table :railway_ipc_published_messages, id: false do | t |
+      t.uuid :uuid, null: false
```

Comment:
So two ways to change primary key from ID to UUID I think. We can just set `id: false` and then use `self.primary_key = "uuid"` or we can specify that the UUID column is the primary key in the migration:

```ruby
create_table :railway_ipc_published_messages, primary_key: 'uuid' do | t |
  ...
end
```

Which generates the SQL:

```sql
CREATE TABLE railway_ipc_published_messages (
  uuid  DEFAULT NULL auto_increment PRIMARY KEY,
  ...
)
```
If we go with the first option, the one we're currently using, what SQL does it generate/is the primary key set on the DB level or just in code somehow? I'm a little confused as to whether or not there is a difference. 
docs here btw https://apidock.com/rails/v4.2.1/ActiveRecord/ConnectionAdapters/SchemaStatements/create_table

Reply:
Talked with @StevenNunez, decided to NOT indicate PK on DB level b/c we DON'T want auto-incrementing/generated uuids--UUID should always come in from RabbitMQ message.